frontend-customers/src/app/pages/success-page/success-page.component.ts

```diff
@@ -0,0 +1,39 @@
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+    selector: 'app-success-page',
+    templateUrl: './success-page.component.html',
+    styleUrls: ['./success-page.component.scss']
+})
+export class SuccessPageComponent implements OnInit {
+
+    start: string | undefined;
+    destination: string | undefined;
+    departureDate1: Date | undefined;
+    arrivalDate1: Date | undefined;
+    departureDate2: Date | undefined;
+    arrivalDate2: Date | undefined;
```

Comment:
Die Daten sollten hier in einem  Array von Objekten gespeichert werden:

z.B
`flights = [
   { name: 'Basel', departureDate: new Date(), arrivalDate: new Date() },
   { name: 'Porto', departureDate: new Date(), arrivalDate: new Date() },
]`